docs/developer-guide.md

```diff
@@ -39,6 +39,38 @@ In general we perfer **Squash and merge** against `v3` or feature branches. When
 
     **At this stage, changes to ideal output, config, error message and line number are not considered breaking*
 
+## Triage Process
+
+We are happy to accept small fixes and small enhancements through pull requests directly.
+For proposals or large changes, we use the following process to pickup, review and approve issues that aligns with our [roadmap](roadmap.md) and priority:
+
+1. Create an issue on GitHub to start a discussion of the proposal.
+
+2. We assign a team member when an issue is picked up in review meetings.
+
+3. Assignee label the issue as `needs-discussion` for group discussion in review meetings.
+
+4. Assignee label the issue as `read-for-review` for approval in review meetings when the following conditions are met:
+
+    - Contains enough details for someone else to start work on it.
+    - Contains work items to support the end to end scenario, including internal works needed beyond docfx.
+    - No open questions.
+    - A draft API proposal to illustrate the change if applicable.
+
+Running the review meeting:
+
+1. Go through issues without assignees that is not labled as future. Assign a team member if the issue is ready to pickup, otherwise lable it as `future`. If the issue has a due date, assign a milestone. We usually have milestones for the next 2 quarters. Issues with assignees can start at any time based on priority and bandwidth.
+
+2. Go through issues with the `ready-for-review` label. Remove the `ready-for-review` label to indicate that reviewing is done. Add `approved` label to approve the design so it is ready to accept pull requests.
```

Comment:
>ready-for-review [](start = 31, length = 16)

shall we add a new label `to-review` to indicate this state? I'm just a bit worried that the review meeting may take too long to go through the list without digging into details. We could probably set up other dedicate meetings to review the items with `to-review` and set `approved` once they are reviewed.